proposals/lip-0043.md

```diff
@@ -149,24 +149,10 @@ sidechainRegParams = {
         },
         "sidechainValidators": {
             "type": "array",
+            "items": ...blsKeyAndBftWeightSchema,
```

Comment:
Minor thing, but for consistency I would keep the `item` property in the last position (after `fieldNumber`)